nodescraper/plugins/inband/kernel/kernel_collector.py

```diff
@@ -73,7 +76,7 @@ def collect_data(
         """
 
         kernel = None
```

Comment:
this extra var can be removed and just use either kernel_info or kernel_version to check on line 113